### ISSUE TITLE
Update the guidance on calculating stream latency

### ISSUE THREAD

**Issue by dturner** [google/oboe#671]
From Thomas: 

"using (frames written - frames read) * (ns per frame) to calculate a delay yielded better results than the suggested way from Don's SO answer. https://stackoverflow.com/a/48773125

Both methods work fine using wired headphones, but when using Bluetooth, the getTimestamp method ends up drifting out of alignment some 150-200ms before periodically resetting."

**Comment by dturner** [comment]
From Thomas: 

The (written-read) method works well for one set of BT headphones, but not at all on a different set.

I've tested both of those headsets with the native YT app, and on a Chromebook, and they didn't have any issues.

The problem seems to be audio drifting out of sync, rather than being a fixed delay behind. If you pause and unpause the video, it is synced up for a few seconds, then drifts apart again.

I've tried multiple ways of calculating delays, but none has worked so far."

@philburk to follow up with Thomas
